utils/dataset_manifest/__init__.py

```diff
@@ -0,0 +1,4 @@
+# Copyright (C) 2021 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+from .core import prepare_meta, VideoManifestManager, ImageManifestManager
```

Comment:
I will vote for renaming `prepare_meta`. I don't like the module name. There are multiple variants like dataset_manifest, meta_info, meta_data, etc.

Reply:
Also it makes sense to move prepare_meta, VideoManifestManager, ImageManifestManager into the module and use dataset_manifest inside core.

Reply:
I'm not sure that I understand correctly. 
1) `prepare_meta` - It is a function that responsible for preparing meta information, which later is used to create a manifest. Why it should be `dataset_manifest`, `meta_info` ...  ?